ui/src/components/WifiConnect.vue

```diff
@@ -90,11 +90,11 @@
             filled
             hide-bottom-space
             :rules="[(val: string) =>
-            val.length > 7
+            (val.length === 0 || val.length > 7)
             || $t('components.wifi.connect.invalid_password_length')]"
             :label="$t('general.password')"
             type="password"
-            :disable="isWifiStatus"
+            :disable="isWifiStatus || wifiConnection?.value?.conn_type === 'NONE'"
```

Comment:
This isn't working, any idea why @maggie0002?

Reply:
It may not need the `.value` component, that's usually only when referring to `ref` in typescript rather than Vue.